server/src/main/java/org/opensearch/index/compositeindex/datacube/MetricStat.java

```diff
@@ -21,7 +21,8 @@ public enum MetricStat {
     AVG("avg"),
     SUM("sum"),
     MIN("min"),
-    MAX("max");
+    MAX("max"),
+    UNSUPPORTED("unsupported");
```

Comment:
I think we need to remove this, this contains the list of supported metrics - this is exposed to the user